src/cmd/servo.rs

```diff
@@ -138,7 +142,16 @@ pub fn run_list(args: &ArgsList) -> Result<()> {
         println!("{}", keys.join(" "));
         return Ok(());
     }
-    println!("{}", list);
+    if args.json {
+        println!("{}", list);
+        return Ok(());
+    }
+    println!("usb_sysfs_path\tserial\tproduct");
+    let mut devices = list.devices().clone();
+    devices.sort_by(|l, r| l.usb_sysfs_path().cmp(r.usb_sysfs_path()));
```

Comment:
I guess we only sort the result for human friendly format but not for json.
If it doesn't hurt to have them sorted in both cases, it might be better to sort them when you create the list in ServeList. Although, I'm not sure even for human readable cases, what order is the best way to present this kind of information. (This is just a comment.)

Reply:
Made the order stable for all the output options.